src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/EventOutcome.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class EventOutcome (
+    val eventOutcome: String? = null,
```

Comment:
think `eventOutcome` and `performance` should be not nullable since the original fields were `final`?